using-timescaledb/data-retention.md

```diff
@@ -147,72 +69,38 @@ SELECT remove_retention_policy('conditions');
 
 The scheduler framework also allows one to view scheduled jobs:
 ```sql
-SELECT * FROM timescaledb_information.drop_chunks_policies;
+SELECT * FROM timescaledb_information.policy_stats;
 ```
 
 For more information, please see the [API documentation][add_retention_policy].
 
+### Data Retention with Continuous Aggregates
 
-### Using External Job Schedulers
-
-While the built-in scheduling framework automates data retention, the
-`drop_chunks` command can be combined with an external tool for job scheduling,
-like `crontab` or `systemd`, to schedule such commands.
-
-#### Using Crontab
-
-The following cron job will drop chunks every day at 3am:
-
-```bash
-0 3 * * * /usr/bin/psql -h localhost -p 5432 -U postgres -d postgres -c "SELECT drop_chunks('conditions', INTERVAL '24 hours');" >/dev/null 2>&1
-```
-
-The above cron job can easily be installed by running `crontab -e`.
-
-
-#### Using a Systemd Timer
-
-On a systemd-based OS (most modern Linux distributions), a systemd [unit][] with
-accompanying unit [timer][] can also be used to implement a
-retention policy.
-
-First, create, e.g., `/etc/systemd/system/retention.service` unit file:
-
-```ini
-[Unit]
-Description=Drop chunks from the 'conditions' table
-
-[Service]
-Type=oneshot
-ExecStart=/usr/bin/psql -h localhost -p 5432 -U postgres -d postgres -c "SELECT drop_chunks('conditions', INTERVAL '24 hours');"
-```
-
-Then create a timer to run this unit, e.g., `/etc/systemd/system/retention.timer`:
-
-```ini
-[Unit]
-Description=Run data retention at 3am daily
-
-[Timer]
-OnCalendar=*-*-* 03:00:00
-
-[Install]
-WantedBy=timers.target
-```
-
-Once these units have been created, run
+Extra care must be taken when using rention policies or `drop_chunks` calls on
+hypertables which have [continuous aggregates][continuous_aggregates] defined on
+them. Any chunks which are dropped in the region still being refreshed by the
```

Comment:
```suggestion
them. Similar to a refresh of a materialized view, a refresh on a continuous aggregate will update the aggregate to reflect changes in the underlying source data. This means that any chunks which are dropped in the region still being refreshed by the
```

Reply:
I really liked how you captured this, added